exporter/coralogixexporter/factory.go

```diff
@@ -19,34 +19,49 @@ import (
 
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/config/configcompression"
 	"go.opentelemetry.io/collector/config/configgrpc"
+	"go.opentelemetry.io/collector/consumer"
 	"go.opentelemetry.io/collector/exporter/exporterhelper"
 )
 
 // NewFactory by Coralogix
 func NewFactory() component.ExporterFactory {
 	return component.NewExporterFactory(
-		typestr,
+		typeStr,
 		createDefaultConfig,
 		component.WithTracesExporter(createTraceExporter),
+		component.WithMetricsExporter(createMetricsExporter),
 	)
 }
 
 func createDefaultConfig() config.Exporter {
 	return &Config{
-		ExporterSettings:   config.NewExporterSettings(config.NewComponentID(typestr)),
-		QueueSettings:      exporterhelper.NewDefaultQueueSettings(),
-		RetrySettings:      exporterhelper.NewDefaultRetrySettings(),
-		TimeoutSettings:    exporterhelper.NewDefaultTimeoutSettings(),
+		ExporterSettings: config.NewExporterSettings(config.NewComponentID(typeStr)),
+		QueueSettings:    exporterhelper.NewDefaultQueueSettings(),
+		RetrySettings:    exporterhelper.NewDefaultRetrySettings(),
+		TimeoutSettings:  exporterhelper.NewDefaultTimeoutSettings(),
+		// Traces GRPC client
 		GRPCClientSettings: configgrpc.GRPCClientSettings{Endpoint: "https://"},
-		PrivateKey:         "",
-		AppName:            "",
+		Metrics: configgrpc.GRPCClientSettings{
+			Endpoint: "https://",
+			Headers:  map[string]string{},
+			// Default to gzip compression
+			Compression: configcompression.Gzip,
+			// We almost read 0 bytes, so no need to tune ReadBufferSize.
+			WriteBufferSize: 512 * 1024,
```

Comment:
I guess this comments comes from here? https://github.com/open-telemetry/opentelemetry-collector/blob/f477970736da2c7911cfeac3107cca52b140acff/exporter/otlpexporter/factory.go#L53 Does it also apply to Coralogix?

Reply:
We are still expermenting with compression and "good" buffer size, so for now, I'll just remove the comment, thanks for your note :)